precovery/main.py

```diff
@@ -91,21 +96,24 @@ def precover_worker(
 
 
 def precover(
-    orbit: AdamOrbits,
+    orbit: Orbits,
     database_directory: str,
     tolerance: float = 1 / 3600,
     start_mjd: Optional[float] = None,
     end_mjd: Optional[float] = None,
     window_size: int = 7,
     allow_version_mismatch: bool = False,
     datasets: Optional[set[str]] = None,
-) -> Tuple[PrecoveryCandidatesQv, FrameCandidatesQv]:
+    propagator: Optional[
+        Propagator
+    ] = None,  # should we initialize an assist propagator?
```

Comment:
```suggestion
    propagator: Optional[Type[Propagator]] = None
```
We should accept a class, rather than an instance. This makes it easier to do the multiprocessing, since we want to make sure we aren't re-using instances of the propagator across threads.